docs/api/configuration.md

```diff
@@ -0,0 +1,64 @@
+# Configuration
+
+Description: TODO.
+
+### Options
+
+- `appium` (`Object?`):
+  - `baseUrl` (`String?`): Base URL of Appium server. Defaults to "http://localhost:4723/wd/hub".
+  - `username` (`String?`): Username for server authentication - useful when connecting to SauceLabs and BrowserStack.
+  - `password` (`String?`): Password for server authentication - useful when connecting to SauceLabs and BrowserStack.
```

Comment:
Not just limited to those platforms. Maybe it should be more generic like "cloud services such as SauceLabs or BrowserStack"